.github/resources/.actions/cli-runner/action.yml

```diff
@@ -0,0 +1,130 @@
+name: Slack CLI Installation and Command Runner 
+description: Download and cache the Slack CLI and run the input command 
+
+inputs:
+  command:
+    description: "Slack CLI command to run"
+    type: string
+    default: ""
+    required: true
+  verbose:
+    description: "Verbose flag"
+    type: boolean
+    default: false
+    required: false
+  cli_version:
+    description: "CLI Version"
+    type: string
+    default: "latest"
+    required: false
+  app_id: 
+    description: "App ID"
+    type: string
+    default: ""
+    required: false
+
```

Comment:
```suggestion
```

🪓 question: For now is this alright to remove? I believe it might conflict with a separate `--app` flag?